nordicsemi/dfu/dfu_transport_ble.py

```diff
@@ -220,10 +219,10 @@ def verify_stable_connection(self):
 
         """
         self.conn_handle = self.evt_sync.wait('connected')
-        if self.conn_handle == DFUAdapter.SUPPORTED_CONN_HANDLE:
+        if self.conn_handle is not None:
             retries = DFUAdapter.CONNECTION_ATTEMPTS
             while retries:
-                if not self.evt_sync.wait('disconnected', timeout=1) == DFUAdapter.SUPPORTED_CONN_HANDLE:
+                if self.evt_sync.wait('disconnected', timeout=1) is None:
                     break
 
                 logger.warning("Received 0x3e, trying to re-connect to: {}".format(self.target_device_addr))
```

Comment:
Can we know for sure that 0x3e was received, or is it just assumed. I see no check for error code 0x3e

Reply:
It is assumed.
Should I add the disconnect reason in the notify-data, so that it can be checked in this function?

Reply:
Or just rewrite the text we assume, instead of claiming as fact